### DISCUSSION TITLE
Welcome to atlas Discussions!

### DISCUSSION THREAD

**Discussion by anirudhRowjee** [anirudhRowjee/atlas#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anirudhRowjee** [comment]
Welcome! This seems like a nice place to discuss new features.

**Comment by Samyak2** [comment]
Hi! This is indeed a nice place to discuss. Looks like it has separate threads even in the same conversation which is really cool!

**Reply by anirudhRowjee** [reply]
Awesome!
